src/main/scala/castalia/JsonConverter.scala

```diff
@@ -0,0 +1,31 @@
+package castalia
+
+import java.io.FileNotFoundException
+import java.net.URL
+
+import spray.json._
+
+import scala.util.Try
+
+/**
+  * Created by jens on 13-12-15.
+  */
+object JsonConverter {
+  def parseJson[T: JsonReader](filename: String): T = {
+    val resource: URL = getClass.getResource("/" + filename)
+    resource match {
+      case url : URL => unmarshalJsonToClass[T](url) getOrElse
+        (throw new UnmarshalException(s"Type could not be unmarshelled from $url"))
+      case _ => throw new FileNotFoundException(filename)
```

Comment:
Hmm.. Apparently it does work:

```
scala> import java.net.URL
import java.net.URL

scala> def pmatch(url : URL) = url match { case u : URL => s"I'm a URL $u"; case any => s"what am I? any" }
pmatch: (url: java.net.URL)String

scala> pmatch(new java.net.URL("http://google.com"))
res8: String = I'm a URL http://google.com

scala> pmatch(null)
res9: String = what am I? any

scala> val u : URL = null
u: java.net.URL = null

scala> pmatch(u)
res10: String = what am I? any
```

The second case it matched on `null`. I guess this is due to type erasure